beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -3836,6 +3836,8 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         }
 
         if let Some(data_columns) = data_columns {
+            // TODO(das): `available_block includes all sampled columns, but we only need to store
+            // custody columns. To be clarified in spec.
```

Comment:
It was discussed in the last breakout call, but there hasn't been any PRs created yet. 
This shoudn't block interop though, because peers should never query for non custody columns.